src/themes/material/Style.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { CssBaseline } from '@material-ui/core'
+
+let Style = () => (
+  <>
+    <style>{`
+      th span {
```

Comment:
This seems like a very generic selector if we assume there are other tables in an app using material

Reply:
just to note - it would be nice to use material's styling API for this, but as far as I know the `th span` inside the material table isn't accessible via its custom selector stuff (since it's actually part of ResultTable).

emotion will make things nicer, but for now I think the solution is just to stick a className on there and style it in the root like we do with grey vest.